module/module.go

```diff
@@ -277,14 +359,62 @@ func readDeps(jsonDeps map[string][]string) []Dependency {
 	return deps
 }
 
+func readPackageInfo(
+	packageRoot string,
+	baseDirectory string,
+	targetGenDir string,
+	className string,
+	instanceDirectory string,
+	jsonConfig *JSONClassConfig,
+	dependencies []Dependency,
+) (*PackageInfo, error) {
+	qualifiedClassName := strings.Title(camelCase(className))
+	qualifiedInstanceName := strings.Title(camelCase(jsonConfig.Name))
+	defaultAlias := camelCase(strings.ToLower(qualifiedInstanceName)) +
+		qualifiedClassName
+
+	relativeGeneratedPath, err := filepath.Rel(baseDirectory, targetGenDir)
```

Comment:
This can be computed once in the module system. If it is then this function doesn't have to return an `error`.